homeassistant/helpers/entity_registry.py

```diff
@@ -52,11 +54,6 @@
 EVENT_ENTITY_REGISTRY_UPDATED = "entity_registry_updated"
 SAVE_DELAY = 10
 _LOGGER = logging.getLogger(__name__)
-DISABLED_CONFIG_ENTRY = "config_entry"
-DISABLED_DEVICE = "device"
-DISABLED_HASS = "hass"
-DISABLED_INTEGRATION = "integration"
-DISABLED_USER = "user"
```

Comment:
This is breaking for custom integrations.

Reply:
it is not.. they are listed/move to lower.

Reply:
![image](https://user-images.githubusercontent.com/195327/144427024-e8d78c85-8202-41ba-80f0-05998dc90ecb.png)

Reply:
Huh... I did not see those earlier :thinking: 
Oh well, carry on :+1: